docs/collector.mssql.md

```diff
@@ -112,138 +112,138 @@ Name | Description | Type | Labels
 `wmi_mssql_dbreplica_log_compression_cachemisses` | _Not yet documented_ | counter | `instance`, `replica`
 `wmi_mssql_dbreplica_log_compressions` | _Not yet documented_ | counter | `instance`, `replica`
 `wmi_mssql_dbreplica_log_decompressions` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_dbreplica_log_remaining_for_undo` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_dbreplica_log_send_queue` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_dbreplica_mirrored_write_transactions` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_dbreplica_recovery_queue_records` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_dbreplica_redo_blocks` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_dbreplica_redo_remaining_bytes` | _Not yet documented_ | counter | `instance`, `replica`
-`wmi_mssql_dbreplica_redone_bytes` | _Not yet documented_ | counter | `instance`, `replica`
+`wmi_mssql_dbreplica_log_remaining_for_undo` | The amount of log, in kilobytes, remaining to complete the undo phase | counter | `instance`, `replica`

```

Comment:
Is this actually exposed as kilobytes, or is it bytes? Most things in the raw data we expose is actually bytes, even if MS converts to larger units in their presentation.